presto-main/src/main/java/com/facebook/presto/operator/aggregation/AggregationCompiler.java

```diff
@@ -224,15 +260,26 @@ private static Method getCombineFunction(Class<?> clazz, Class<?> stateClass)
         return inputFunctions;
     }
 
-    private List<Type> getInputTypes(Method inputFunction)
+    private static List<TypeSignature> signaturesFromTypes(List<Type> types)
```

Comment:
move it to BindableAggregationFunction. This is the only place where it is used.